lib/rialto/etl/cli/publications.rb

```diff
@@ -1,27 +1,114 @@
 # frozen_string_literal: true
 
-require 'csv'
 require 'fileutils'
-require 'rialto/etl/cli/composite_etl'
+require 'jsonpath'
 
 module Rialto
   module Etl
     module CLI
-      # Publications iterates over the people in the input file CSV and for each
-      # calls extract, transform, and load.
-      class Publications < CompositeEtl
-        protected
+      # Publications grabs all Stanford University-affiliated publications from
+      # the WoS API, then transforms and loads them.
+      class Publications < Thor
```

Comment:
Not with `exe/publications`, no. There is if you use `exe/{extract,transform,load}`. And there will be once we implement the ability to load pubs loaded/updated since a given date. Are there other possible incantations you think `exe/publications` ought to support in the meantime?